packages/nimble-components/src/rich-text/editor/tests/rich-text-editor.spec.ts

```diff
@@ -1788,7 +1788,7 @@ describe('RichTextEditor', () => {
 
         expect(inputEventListener.spy).toHaveBeenCalledTimes(1);
 
-        await pageObject.setEditorTextContent('');
```

Comment:
This was resulting in the warning:

> WARN: '[tiptap warn]: Invalid content.', 'Passed value:', Object{type: 'text', text: ''}, 'Error:', RangeError: Empty text nodes are not allowed
RangeError: Empty text nodes are not allowed

The test description states removved text should trigger the event so used the same remove text pattern as the following test